src/modules/home/Home.js

```diff
@@ -842,25 +866,30 @@ const Home = (props) => {
         onPress={(name, position) => dialogClickHandler('notebookPanelMenuVisible', name, position)}
         onTouchOutside={() => toggleDialog('notebookPanelMenuVisible')}
       />
-      <Modal
-        isVisible={props.isImageModalVisible}
-        useNativeDriver={true}
-        style={{flex: 1}}
-      >
-        <View style={homeStyles.modal}>
-          <Button
-            type={'clear'}
-            titleProps={{color: 'white'}}
-            title='Hide modal'
-            onPress={() => toggleImageModal()}/>
-          <Image
-            source={props.selectedImage
-              ? {uri: useImages.getLocalImageSrc(props.selectedImage.id)}
-              : require('../../assets/images/noimage.jpg')}
-            style={{width: wp('90%'), height: hp('90%')}}
-          />
-        </View>
-      </Modal>
+      {props.isImageModalVisible && (
+        <SafeAreaView>
+          <ScrollView>
+            <FlatListSlider
+              data={imageSlideShowData}
+              imageKey={'uri'}
+              autoscroll={false}
+              separator={0}
+              loop={true}
+              width={props.deviceDimensions.width}
+              height={props.deviceDimensions.height}
+              onPress={(item) => {
```

Comment:
How about if on press of the picture the 'X' and other option buttons go away so all you see is the picture? Then you can continue scrolling through with just seeing a full page image and no button overlays. But if you then press on the image the buttons will show up again. (This is how it is in the Google Photos app.)